data/tlg0010/tlg016/tlg0010.tlg016.perseus-grc2.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-  <teiHeader>
+  <teiHeader xml:lang="eng">
     <fileDesc>
       <titleStmt>
-        <title>Archidamus</title>
-        
+        <title xml:lang="grc">Αρχίδαμος</title>
```

Comment:
Ἀρχίδαμος

Reply:
Well and to think I tried using an unnamed AI tool to translate titles into Greek! I won't do that again @lcerrato and I thought I had got all the printings. Glad I did this pull request in smaller batches at least.

Reply:
The Perseus TOC typically has them (http://www.perseus.tufts.edu/hopper/text?doc=Perseus:text:1999.01.0143 -- albeit there are a couple of errors here). Wikipedia is also better for ancient title. 
The tool missed all breathing marks. You mainly have modern Greek rather than ancient.

Reply:
This was my omission, so I am happy to have issues opened and do a pass just on the correct titles. I think for these large reviews we should be working thematically anyhow.